src/pages/home/report/ReportActionItem.tsx

```diff
@@ -397,7 +398,7 @@ function ReportActionItem({
     const mentionReportContextValue = useMemo(() => ({currentReportID: report?.reportID ?? '-1'}), [report?.reportID]);
 
     const actionableItemButtons: ActionableItem[] = useMemo(() => {
-        if (ReportActionsUtils.isActionableAddPaymentCard(action) && shouldRenderAddPaymentCard()) {
+        if (ReportActionsUtils.isActionableAddPaymentCard(action) && !doesUserHavePaymentCardAdded() && shouldRenderAddPaymentCard()) {
```

Comment:
`doesUserHavePaymentCardAdded` depends on `userBillingFundID` however `userBillingFundID` isn't included in useMemo dependencies array, which caused [this issue](https://github.com/Expensify/App/issues/52564). More details: https://github.com/Expensify/App/issues/52564#issuecomment-2478148847